subprojects/docs/src/docs/release/notes.md

```diff
@@ -169,9 +117,75 @@ In addition, this behavior can be enabled from the command line for individual b
 
 More information is available in the [Java Plugin documentation for test execution](userguide/java_plugin.html#sec:test_execution).
 
-<!--
-### Example new and noteworthy
--->
+### Allow declared reasons for dependency and resolution rules
+
+In complex builds, it can become hard to interpret dependency resolution results and why a [dependency declaration](userguide/customizing_dependency_resolution_behavior.html) or a [rule](userguide/inspecting_dependencies.html#sec:dependency_declaration_reasons) was added to a build script. To improve on this situation, we extended all the corresponding APIs with the capability to define a _reason_ for each declaration or rule. 
+
+These reasons are shown in dependency insight reports and error messages if the corresponding declaration or rule influenced the resolution result. In the future, they will also be shown in build scans.
+
+    dependencies {
+        implementation('org.ow2.asm:asm:6.0') {
+            because 'we require a JDK 9 compatible bytecode generator'
+        }
+    }
+    
+### BOM import
+
+Gradle now [provides support](userguide/managing_transitive_dependencies.html#sec:bom_import) for importing [bill of materials (BOM) files](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies), which are effectively `.pom` files that use `<dependencyManagement>` to control the dependency versions of direct and transitive dependencies. It works by declaring a dependency on a BOM.
+
+    dependencies {
+        implementation('org.springframework.boot:spring-boot-dependencies:1.5.8.RELEASE') {
+            because 'it defines dependency versions for my application'
+        }
+    
+        implementation 'com.google.code.gson:gson'
+        implementation 'dom4j:dom4j'
+    }
+
+Here, for example, the versions of `gson` and `dom4j` are provided by the Spring Boot BOM.
+
+**Note:** This is a _Gradle 5.0 feature preview_, which means it is a potentially breaking change that will be activated by default in Gradle 5.0. It can be turned on in Gradle 4.6+ by adding `enableFeaturePreview('IMPROVED_POM_SUPPORT')` in _settings.gradle_.
+
+### Support for optional dependencies in POM consumption
```

Comment:
Do we have any documentation that we can link to? What things does a user need to think about when migrating their Maven build to Gradle?

Reply:
Added a link